### ISSUE TITLE
music after desktop start is too repetitive and annoying

### ISSUE THREAD

**Issue by hw-claudio** [helloSystem/Utilities#50]
a more calm theme should be found, needs to be more mellow,
less repetitive, and possibly at a lower volume by default.

**Comment by hw-claudio** [comment]
If going for a simple one-off, not repeated sound, some jingle like:

https://www.storyblocks.com/audio/stock/magic-turning-spells-casting-rleiye32uvrk0wxrx31.html

would do; If you want instead to keep the current behavior of having looping music until the user presses continue,
then indeed volume needs to be way lower,

and the theme should be mellow, relaxing. Maybe even using a strings based theme (guitar or so), rather than a too 'exciting' piano loop (i'd avoid fast tempo and fast change below lo and hi tones).

https://www.storyblocks.com/audio/stock/everyone-re7hxgolpkczx1nrf.html

Just to give an idea, not sure if this is really royalty free as they claim, but should give you an idea of what I mean.

**Comment by louies0623** [comment]
This is my suggestion
[https://www.storyblocks.com/audio/stock/follow-your-heart-hlfearahldsk0wy3bhn.html](url)
or
[https://www.storyblocks.com/audio/stock/chill-happy-clapping-whistle-and-ukelele-stdiyqwqvkhma64d9.html](url) <<LIKE

**Comment by hw-claudio** [comment]
I think that the tunes are nice, but don't fit the requirements for me,
the first has very fast high/low contrast, the second with the whistles is just too exciting.

The problem is that the music is sustained until the user presses Continue, ... etc.
Potentially the user can linger there for a long time until he/she figures out how to interact.

The audio input should not become an obstacle in trying to figure out what to do, this was my experience.

**Comment by louies0623** [comment]
Ok let me find

**Comment by louies0623** [comment]
I can't seem to find a better one 😅

**Comment by louies0623** [comment]
https://www.storyblocks.com/audio/stock/gather-around-the-fire-instrumental-version-bttwe_c9dki08n2jb.html

**Comment by hw-claudio** [comment]
Yes, I think the second one "gather around the fire" goes in the right direction, especially at a low volume.

**Comment by probonopd** [comment]
These things are very subjective of course but I was looking for something a little bit (but not overly) more "computer-y"

**Comment by louies0623** [comment]
Seriously, I want to try to find music in the style of this film in FMA, but my ability is limited.
https://youtu.be/apx_HyxFqjA
But this is only suitable for the welcome intro.

**Comment by louies0623** [comment]
Welcome music in the new beta, I think the sound is too low
Restore. As for the loop part, I suggest canceling it, and let him gradually mute the whole song during playback.

**Comment by probonopd** [comment]
Increased volume again a bit:
https://github.com/helloSystem/Utilities/commit/c993a84f25cefa13660393ede7ec0efffb910f0f

**Comment by grahamperrin** [comment]
Was there a slightly different soundtrack when the sound was first introduced? If so, I preferred that track. 

For me, the repetitiveness comes only after repeatedly booting the live system (with non-persistent storage for preferences etc.).

**Comment by probonopd** [comment]
No, never changed the music...
